.concourse/pipeline.yaml.gomplate

```diff
@@ -340,24 +340,18 @@ jobs:
     {{- end }}
     config:
       platform: linux
-      image_resource:
+      image_resource: &catapult-image
         type: registry-image
         source:
-          repository: thulioassis/bazel-docker-image
-          tag: 2.0.0
+          repository: splatform/catapult
+          tag: bedde80dff4cd8c296c5c9774f83a9a191a963af
       inputs:
       - name: kubecf-{{ $branch }}
       run:
-        path: "/bin/bash"
+        path: "/usr/bin/make"
         args:
-        - -ce
-        - |
-          cd kubecf-{{ $branch }}
-          ./dev/linters/shellcheck.sh
-          ./dev/linters/yamllint.sh
-          ./dev/linters/helmlint.sh
-          bazel test //rules/kubecf:create_sample_values_test
-          bazel test //deploy/helm/kubecf:values_doc_test
+        - kubecf-{{ $branch }}
```

Comment:
You're missing the `-C` to change directory, I think.

Reply:
Correct, thanks!